RELEASE/scripts/autoscend/iotms/mr2022.ash

```diff
@@ -145,6 +145,52 @@ boolean auto_fightLocketMonster(monster mon, boolean speculative)
 
 }
 
+boolean auto_haveGreyGoose()
+{
+	if(auto_have_familiar($familiar[Grey Goose]))
+	{
+		return true;
+	}
+	return false;
+}
+
+int gooseExpectedDrones()
+{
+	if(!auto_haveGreyGoose()) return 0;
+	if(my_familiar() == $familiar[Grey Goose])
+	{
+		set_property("auto_gooseExpectedDrones", familiar_weight($familiar[Grey Goose]) - 5);
+	}
+	return get_property("auto_gooseExpectedDrones").to_int();
+}
+
+boolean dronesOut() //want a function to override the task order if we have drones out so as not to waste them
+{
+	if(!auto_haveGreyGoose()) return false;
+	if(get_property("gooseDronesRemaining").to_int() > 0)
+	{
+		return true;
+	}
+	return false;
+}
+
+void prioritizeGoose() //prioritize Goose only if we still have things to get
+{
+	if(!auto_haveGreyGoose()) return;
+	if(	internalQuestStatus("questL04Bat") <= 1 ||
```

Comment:
could probably add a handful of others here but these are a great starting salvo. if it would be helpful, i can list out 5-6 other places we might want to think about goose duping

Reply:
@docrostov  lets discuss this in Discord. I want to get this merged as it's been languishing for months.